### PR TITLE
[RFR] Detect the victory

### PR DESCRIPTION
**Pull request by Luwangel**
- [x] Refact the api management to detect the victory
- [x] Change the game id generation
- [x] Display a message to the user in case of victory

### CODE REVIEW

**Comment by Luwangel** [comment]
capture du 2017-11-08 16-41-54

File: app/templates/game.html.twig [hunk 1]
```diff
@@ -2,30 +2,23 @@
 
 {% block header %}   
   <div class="section">
-    <h5>Current game</h5>
+    {% if isVictory %}
+      <h5>Congratulations, you have solved the puzzle !</h5>
```
Comment (jdemangeon) [inline app/templates/game.html.twig:6]: Can you use translation system ?
http://symfony.com/doc/master/translation.html

File: app/templates/game.html.twig [hunk 2]
```diff
@@ -2,30 +2,23 @@
...
-            {% endif %}
-          {% endfor %}
-        </div>
-      {% endfor %}
+    <div class="row">
+      {% include 'grid.html.twig' %}
     </div>
+    {% if isVictory %}
+      <div class="row">
+        <a class="btn" href="/game"><i class="material-icons left">keyboard_return</i>Play another game</a>
```
Comment (jdemangeon) [inline app/templates/game.html.twig]: Use Symfony router functions (path, url, ...)
https://symfony.com/doc/master/routing.html

File: app/templates/game.html.twig (hunk 1 above)
Comment (Luwangel) [reply app/templates/game.html.twig:6]: Unfortunately I'll not have the time to apply this review. I'll keep that for another PR.

**Review by alexisjanvier — APPROVED** [review]

**Pull request merged**